deploy-java/action.yml

```diff
@@ -138,6 +138,19 @@ runs:
         jiraUsername: ${{ inputs.jiraUsername }}
         jiraPassword: ${{ inputs.jiraPassword }}
 
+    - name: On success notify okay to test
+      uses: actions/github-script@v6
+      if: ${{ (inputs.environment !='Production') }}
+      with:
+        github-token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        script: |
+          const result = await github.rest.actions.createWorkflowDispatch({
+            owner: 'cartoncloud',
+            repo: 'api-functional-tests',
+            workflow_id: 'run-tests.yml',
+            environment: ${{ inputs.environment }}
```

Comment:
Will this work? Java services have environments as `QA 01` or `Test 01` whereas Api Functional tests have `qa1`, `test1`

Reply:
The environments are going to be re-created with standardised names